src/components/EditOrganization/index.tsx

```diff
@@ -175,6 +201,17 @@ export default function OrganizationView({ organization, reloadOrganizationData
                 fullWidth
               />
             </Grid>
+          ) : (
+            !isMobile && <Grid item xs={gridSize()} paddingLeft={isMobile ? 0 : theme.spacing(2)} />
+          )}
+          {timeZoneFeatureEnabled && (
+            <Grid item xs={gridSize()}>
+              <TimeZoneSelector
```

Comment:
i think this has a tooltip option, we could pass one with placeholder text.. objective is to show users the implications of setting a time zone

Reply:
The tooltip can also explain the default as well.